StickyTabBarViewController/Classes/StickyViewControllerSupportingTabBarController.swift

```diff
@@ -10,6 +10,7 @@ import UIKit
 public protocol StickyViewControllerSupporting: UITabBarController {
     var collapsedHeight: CGFloat { get set }
     var animationDuration: TimeInterval { get set }
+    var childViewController: Expandable? { get }
```

Comment:
Do you think this is a good name for the sticked view controller while accessing from tab bar controller? I thought it would be consistent with method names. `expandChild`, `collapseChild`

Reply:
I think it is! 💪